### PR TITLE
Update main-ar.json

### PR DESCRIPTION
**Pull request by ali-alhaidary**
updated translation...

### CODE REVIEW

**Comment by ali-alhaidary** [comment]
@damencho

**Comment by ali-alhaidary** [comment]
> Hi, thanks for your contribution! If you haven't already done so, could you please make sure you sign our CLA (https://jitsi.org/icla for individuals and https://jitsi.org/ccla for corporations)? We would unfortunately be unable to merge your patch unless we have that piece :(.

Already signed it.

File: lang/main-ar.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         "noResults": "لم يُعثَر على أي نتيجة بحث متطابقة",
         "outlookEmail": "بريد مايكروسوفت",
         "phoneNumbers": "أضف ارقام هواتف",
-        "search": "ابحث عن أشخاص",
+        "searching": "",
```
Comment (saghul) [inline lang/main-ar.json:23]: Maybe translate this?

File: lang/main-ar.json [hunk 2]
```diff
@@ -357,7 +357,7 @@
         "viewUpgradeOptionsTitle": "لقد اكتشفت ميزة مميزة!",
         "WaitForHostMsg": "لم يبدأ المؤتمر <b>{{room}}</b> بعد. إن كنت المضيف والراعي، فنرجو تأكيد ذلك عبر الاستيثاق أو انتظر وصول المضيف رجاءً. ",
         "WaitForHostMsgWOk": "لم يبدأ المؤتمر <b>{{room}}</b> بعد. إن كنت المضيف والراعي، فاضغط على «تمام» للاستيثاق أو انتظر وصول المضيف رجاءً.",
-        "WaitingForHost": "ننتظر وصول المضيف ...",
+        "WaitingForHostTitle": "",
```
Comment (saghul) [inline lang/main-ar.json:360]: Ditto.

File: lang/main-ar.json (hunk 1 above)
Comment (ali-alhaidary) [reply lang/main-ar.json:23]: Is this new?

Comment (saghul) [reply lang/main-ar.json:23]: Did you run the update script? In that case yeah.

File: lang/main-ar.json (hunk 2 above)
Comment (ali-alhaidary) [reply lang/main-ar.json:360]: "searching": "يبحث",

File: lang/main-ar.json (hunk 1 above)
Comment (ali-alhaidary) [reply lang/main-ar.json:23]: "searching": "يبحث",

File: lang/main-ar.json (hunk 2 above)
Comment (ali-alhaidary) [reply lang/main-ar.json:360]: "WaitingForHostTitle": "في انتظار المضيف ...",
"image1": "شاطئ",

**Comment by ali-alhaidary** [comment]
should I edit the file as well?

**Comment by saghul** [comment]
Yeah please edit those as well while you're at it...

**Comment by ali-alhaidary** [comment]
> Yeah please edit those as well while you're at it...

Done...

**Comment by ali-alhaidary** [comment]
@saghul

**Comment by gpatel-fr** [comment]
@ali-alhaidary FYI about 20 new keys were added to main.json since your last changes

**Comment by ali-alhaidary** [comment]
> @ali-alhaidary FYI about 20 new keys were added to main.json since your last changes

Kindly add them all so that I translate...

**Comment by ali-alhaidary** [comment]
@saghul  @gpatel-fr 
Added and translated all...

**Comment by saghul** [comment]
Yo have deleted the file...

**Comment by ali-alhaidary** [comment]
> Yo have deleted the file...

Really !!! How do I put it back???

**Comment by ali-alhaidary** [comment]
Attached...

**Comment by ali-alhaidary** [comment]
It seems that I deleted the file. I will put new PR

**Pull request closed without merging**